translator/source_code/icone_translator.py

```diff
@@ -307,62 +438,67 @@ def add_ids(message, add_ids):
             #     types_of_work['road_event_id'] = road_event_id
     return message
 
+
 def parse_arguments(argv):
     inputfile = ''
     outputfile = 'wzdx_translated_output_message.geojson'
     try:
-      opts, args = getopt.getopt(argv,"hi:o:",["ifile=","ofile="])
+        opts, args = getopt.getopt(argv, "hi:o:", ["ifile=", "ofile="])
     except getopt.GetoptError:
-     # print ('test.py -i <inputfile> -o <outputfile>')
-      sys.exit(2)
+        # print ('test.py -i <inputfile> -o <outputfile>')
+        sys.exit(2)
     for opt, arg in opts:
-      if opt == '-h':
-       #  print ('test.py -i <inputfile> -o <outputfile>')
-         sys.exit()
-      elif opt in ("-i", "--ifile"):
-         inputfile = arg
-      elif opt in ("-o", "--ofile"):
-         outputfile = arg
-   # print ('Input file is "', inputfile)
-   # print ('Output file is "', outputfile)
-    return inputfile,outputfile
-
-inputfile,outputfile=parse_arguments(sys.argv[1:])
-
-def initialize_info() :
+        if opt == '-h':
+            #  print ('test.py -i <inputfile> -o <outputfile>')
+            sys.exit()
+        elif opt in ("-i", "--ifile"):
+            inputfile = arg
+        elif opt in ("-o", "--ofile"):
+            outputfile = arg
+    # print ('Input file is "', inputfile)
+    # print ('Output file is "', outputfile)
+    return inputfile, outputfile
+
+
+inputfile, outputfile = parse_arguments(sys.argv[1:])
+
+
+def initialize_info():
     info = {}
 
     #### Consider whether this id needs to be hardcoded or generated
-    info['feed_info_id'] = "feed_info_id"
+    info['feed_info_id'] = "104d7746-688c-44ed-b195-2ee948bf9dfa"
```

Comment:
Where is this GUID coming from?

Reply:
This was a randomly generated UUID. The WZDx feed_info_id should remain the same each time a feed is updated, to show that we are updating a previous data feed and not creating an entirely new one. We assumed that we would always be updating the same feed, since we will always be drawing from the same data source. 
If this is not the case, then we can alter this logic to randomly generate a new UUID, but then we would always be generating a new WZDx feed.

Reply:
Ah I see, that makes sense now. Thanks for the clarification! I think for now at least we plan to continue consuming from the same data feed and can leave this id alone.